src/main/resources/assets/farmersdelight/lang/es_es.json

```diff
@@ -53,122 +58,152 @@
 
   "item.farmersdelight.tomato_sauce": "Salsa de tomate",
   "item.farmersdelight.fried_egg": "Huevo frito",
-  "item.farmersdelight.milk_bottle": "Frasco de leche",
+  "item.farmersdelight.milk_bottle": "Botella de leche",
   "item.farmersdelight.hot_cocoa": "Chocolate caliente",
+  "item.farmersdelight.wheat_dough": "Masa de trigo",
   "item.farmersdelight.raw_pasta": "Pasta cruda",
-  "item.farmersdelight.pumpkin_slice": "Pedazo de calabaza",
-  "item.farmersdelight.cabbage_leaf": "Hoja de repollo",
+  "item.farmersdelight.pumpkin_slice": "Rebanada de calabaza",
+  "item.farmersdelight.cabbage_leaf": "Hoja de col",
   "item.farmersdelight.minced_beef": "Carne molida",
-  "item.farmersdelight.beef_patty": "Hamburgesa cosida",
-  "item.farmersdelight.chicken_cuts": "Pedazos de pollo crudo",
-  "item.farmersdelight.cooked_chicken_cuts": "Pedazos de pollo cocinado",
-  "item.farmersdelight.cod_slice": "Pedazos de bacalao crudo",
-  "item.farmersdelight.cooked_cod_slice": "Pedazos de bacalao cocinado",
-  "item.farmersdelight.salmon_slice": "Pedazos de salmon crudo",
-  "item.farmersdelight.cooked_salmon_slice": "Pedazos de salmon cocinado",
-
-  "item.farmersdelight.pie_crust": "Corteza para tarta",
-  "item.farmersdelight.cake_slice": "Pedazo de tarta",
-  "item.farmersdelight.apple_pie_slice": "Pedazo de tarta de manzana",
-  "item.farmersdelight.sweet_berry_cheesecake_slice": "Pedazo de cheesecake de baya dulce",
-  "item.farmersdelight.chocolate_pie_slice": "Pedazo de tarta de chocolate",
-  "item.farmersdelight.sweet_berry_cookie": "Galleta de baya dulce",
+  "item.farmersdelight.beef_patty": "Empanada de ternera",
```

Comment:
This one seems to suggest a baked good, rather than a beef patty; the previous one might be more accurate.

Reply:
Hello! how are you? I hope you are well, yes, I have checked for the second time more carefully and found some errors. Thanks for the tip!